### PR TITLE
Set `SameSite=Lax` on screenResultion cookie

### PR DESCRIPTION
**Pull request by StefanSpieker**
Set `SameSite=Lax` on screenResultion cookie. I also replaced the YUI function with native javascript, since the YUI component was not aware of this newer attribute. The reason for adding this was a Firefox warning in the console. 

I found this nice explanation what the attribute does: [samesite-cookies-explained](https://web.dev/i18n/en/samesite-cookies-explained/) Although the cookie functionality will not break with the upcoming changes, I thought we can also mark this as stric, because we do not need it somewhere else or even in a third party website.

### Testing done

I started with the new javascript and checked that the cookie and compared the new and the old cookie that the only difference is the new attribute:

old:
grafik

new:
grafik

### Proposed changelog entries

- Set `SameSite=Lax` on screenResultion cookie

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7417"></a>

### CODE REVIEW

**Comment by daniel-beck** [comment]
Why did you choose `Strict`? This seems wrong to me based on the distinction described on https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite: We would want the cookie to be sent when navigating to Jenkins from elsewhere, so that Jenkins can generate graphs etc. in the correct resolution on the first request. Or is that not needed because all consumers of the cookie are in secondary (img etc.) requests?

**Comment by StefanSpieker** [comment]
@daniel-beck You are right, on a second time reading the documentation, I guess `Lax` is a better setting - it is just the screen resolution

**Review by timja — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]

File: core/src/main/resources/hudson/model/View/screen-resolution.js
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/View/screen-resolution.js:12]: Unnecessary template literal?

**Review by daniel-beck — APPROVED** [review]
Looks reasonable. Untested.

File: core/src/main/resources/hudson/model/View/screen-resolution.js
Comment (StefanSpieker) [reply core/src/main/resources/hudson/model/View/screen-resolution.js]: Yes, thanks

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
